jadx-plugins-tools/src/main/java/jadx/plugins/tools/JadxPluginsTools.java

```diff
@@ -58,6 +58,7 @@ public JadxPluginMetadata resolveMetadata(String locationId) {
 
 	public List<JadxPluginUpdate> updateAll() {
 		JadxInstalledPlugins plugins = loadPluginsJson();
+		plugins.addInstalled(JadxPluginsList.getInstance().get());
```

Comment:
`loadPluginsJson()` already load all installed plugins, so adding anything else not needed.